libs/librepcbcommon/systeminfo.cpp

```diff
@@ -55,11 +61,13 @@ QString SystemInfo::getFullUsername() noexcept
     QString username("");
 
 #if (defined(Q_OS_UNIX) || defined(Q_OS_LINUX)) && (!defined(Q_OS_MACX)) // For UNIX and Linux
-    QString command("grep \"^$USER:\" /etc/passwd | awk -F: '{print $5}'");
-    QProcess process;
-    process.start("sh", QStringList() << "-c" << command);
-    process.waitForFinished(500);
-    username = QString(process.readAllStandardOutput()).remove("\n").remove("\r").trimmed();
+    struct passwd* uid = getpwuid(getuid());
```

Comment:
- I think the "struct" keyword it not required here.
- Why is the variable called `uid`? It's a pointer to a passwd struct, not to a UID...
- The returned pointer could be NULL (in case of an error). This should be checked before accessing any field of the passwd struct.